Samples/V1.0Samples/LocalMediaSamples/AudioVideoPlaybackBot/ADF/bicep/global/region.json

```diff
@@ -0,0 +1,442 @@
+{
+  "switzerlandnorth": {
+    "displayname": "Switzerland North",
+    "location": "switzerlandnorth",
+    "first": "Switzerland",
+    "second": "North",
```

Comment:
same here, can we remove config files if they don't affect the bot functionalities

Reply:
Would have the same reason and follow up as the prefix file. Thanks